pages/pycon-landing.tsx

```diff
@@ -70,88 +75,150 @@ export default function PyConIndiaWeb() {
             <br />
             Stay tuned for more updates!
           </p>
-
-          <div className="mt-10 text-xl flex items-center justify-center">
-            <a
-              href="https://docs.google.com/forms/d/e/1FAIpQLSdmlVnxDBENiXs17hQeUtgqOZdcsWI7lmQpXq5m9K-e0eS52Q/viewform"
-              className="rounded-md bg-sky-900 px-3.5 py-2.5 text-lg font-semibold text-white shadow-lg hover:bg-sky-700 focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
-              target="_blank"
-            >
-              Become a Volunteer
-            </a>
+          <h4 className="mt-10 text-2xl font-bold">Previous Year Websites</h4>
+          <div className="flex justify-center mt-4">
+            <div className="mx-4">
+              <a
+                href="https://in.pycon.org/2016"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-blue-600 underline"
+              >
+                PyCon India 2016 - Bangalore
+              </a>
+            </div>
+            <div className="mx-4">
+              <a
+                href="https://in.pycon.org/2017"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-blue-600 underline"
+              >
+                PyCon India 2017 - Delhi
+              </a>
+            </div>
+            <div className="mx-4">
+              <a
+                href="https://in.pycon.org/2018"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-blue-600 underline"
+              >
+                PyCon India 2018 - Hyderabad
+              </a>
+            </div>
+            <div className="mx-4">
+              <a
+                href="https://in.pycon.org/2019"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-blue-600 underline"
+              >
+                PyCon India 2019 - Chennai
+              </a>
+            </div>
+            <div className="mx-4">
+              <a
+                href="https://in.pycon.org/2019"
```

Comment:
@inovizz 
2020 and 2021 links still point to 2019.

Reply:
Thanks for the catch, fixing it.